common/dependabot-common.gemspec

```diff
@@ -35,8 +35,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "parser", ">= 2.5", "< 4.0"
   spec.add_dependency "toml-rb", ">= 1.1.2", "< 3.0"
 
-  spec.add_development_dependency "debase", "0.2.3"
-  spec.add_development_dependency "debase-ruby_core_source", "0.10.16"
```

Comment:
Note to myself when I put together the "real" PR:
* added in https://github.com/dependabot/dependabot-core/pull/5086/files
* debase doesn't currently support Ruby 3 or 3.1: https://github.com/ruby-debug/ruby-debug-ide/issues/223#issuecomment-1009609192
* ruby 3 ships with a new `debug` gem? don't have many details on it... 
* not sure if we should also remove `ruby-debug-ide` or at least bump the version pin: https://github.com/dependabot/dependabot-core/pull/5086/files#diff-e843e646a7311fb218cf05845818b31d84bc04c0742f0567395c0c289cacf72cR46

but we shouldn't need debase, it's only for debugging and we can always add it back later when they add support...